census_area/core.py

```diff
@@ -47,7 +47,7 @@
         2016: 'https://tigerweb.geo.census.gov/arcgis/rest/services/TIGERweb/tigerWMS_ACS2016/MapServer/26',
         2017: 'https://tigerweb.geo.census.gov/arcgis/rest/services/TIGERweb/tigerWMS_ACS2017/MapServer/26',
         2018: 'https://tigerweb.geo.census.gov/arcgis/rest/services/TIGERweb/tigerWMS_ACS2017/MapServer/28',
-        2019: 'https://tigerweb.geo.census.gov/arcgis/rest/services/TIGERweb/tigerWMS_ACS2017/MapServer/28',
```

Comment:
i think we *do* want census designated spaces. they should be a superset of incorporated places

Reply:
okay, i was wrong. we do want incorporated places. great catch @hancush !

Reply:
let's update the year endpoint too.